app/services/file_appender.rb

```diff
@@ -125,6 +126,11 @@ def create_node(file)
       node
     rescue StandardError => error
       message = "#{self.class}: Failed to append the new file #{original_filename} for #{node.id} to resource #{parent.id}: #{error}"
+      # For FileSet parents we're likely attaching a derivative, so let the
+      # error fall through so sidekiq restarts.
```

Comment:
suggestion: use "raise the error" instead of "let the error fall through"

Reply:
Done